src/ir/function.rs

```diff
@@ -228,6 +228,11 @@ impl FunctionSig {
         let ret = Item::from_ty_or_ref(ty_ret_type, None, None, ctx);
         let abi = get_abi(ty.call_conv());
 
+        if abi == None {
+            assert!(cursor.kind() == CXCursor_ObjCInstanceMethodDecl,
```

Comment:
nit: if you use `assert_eq!` instead, the panic message will also show the failing cursor kind.